lib/util/got/cache-get.js

```diff
@@ -0,0 +1,22 @@
+const crypto = require('crypto');
+const got = require('got');
+
+// global.repoCache is reset to {} every time a repository is initialized
+// With this caching, it means every GET request is cached during each repository run
+
+module.exports = got.create({
+  options: {},
+  handler: (options, next) => {
+    if (options.method === 'GET') {
+      const cacheKey = crypto
+        .createHash('md5')
+        .update('got-' + options.href + JSON.stringify(options.headers))
+        .digest('hex');
+      if (!global.repoCache[cacheKey]) {
+        global.repoCache[cacheKey] = next(options);
+      }
+      return global.repoCache[cacheKey];
```

Comment:
Note that you return the same promise for all requests, so if you do any changes on the response, the change will be global.

Reply:
True. Do you think it would be ok to do a deep copy of `res` before returning it? That would be safer than assuming the layer above always duplicates it or doesn't change it